src/Shouldly/Shouldly.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>net451;net452;netstandard1.3</TargetFrameworks>
```

Comment:
Hi @onovotny 
It was a desperate attempt to get the build working before popping out, given the tests were running against 452. Thanks for the PR @slang25, that's all green now - good job! 👍